docs/trace_collection.md

```diff
@@ -271,6 +271,67 @@ val request = Request.Builder()
 
 **Note**: If you use multiple Interceptors, this one must be called first.
 
+### RxJava
+
+To provide a continuous trace inside a RxJava stream you need to follow the steps below:
+1. Add the [OpenTracing for RxJava][8] dependency into your project and follow the **Readme** file 
+   for instructions. For example for a continuous trace you just have to add:
+   ```kotlin
+   TracingRxJava3Utils.enableTracing(GlobalTracer.get())
+   ```
+2. Then in your project open a scope when the Observable is subscribed and close it when it completes. Any span
+   created inside the stream operators will be displayed inside this scope (parent Span):
+   ```kotlin
+    Single.fromSupplier{}
+        .subscribeOn(Schedulers.io())
+        .map { 
+            val span = GlobalTracer.get().buildSpan("<YOUR_OP_NAME>").start()
+            // ...
+            span.finish()
+        }
+       .doOnSubscribe {
+           val span = GlobalTracer.get()
+               .buildSpan("<YOUR_OP_NAME>")
+               .start()
+           GlobalTracer.get().scopeManager().activate(span)
+       }
+       .doOnTerminate {
```

Comment:
what if downstream unsubscribes before stream terminates, then we will have unfinished span?

Reply:
This is just an example which in normal conditions should work. We cannot cover all the corner cases in these examples and it's in the user responsibility to make sure he's handling those well.

Reply:
yes, completely agree that it is up to the user. my point is that symmetric callback for `doOnSubscribe` will be either `doFinally` or `doOnDispose`

Reply:
Oh...you are definitely right here. I will change it :), not used with all these new Rx operators